migration/394lts-395lts/09940_4176_File_Handler_not_supported_on_System_client.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<Migrations>
+  <Migration EntityType="D" Name="#4176 File Handler not supported on System client" ReleaseNo="3.9.5" SeqNo="9940">
```

Comment:
Please change the ReleaseNo="3.9.5" to ReleaseNo="3.9.4.001"

Reply:
All xml files on this folder are tagged with release 3.9.5 https://github.com/adempiere/adempiere/tree/hotfix/3.9.4.001/migration/394lts-395lts

Reply:
The xml migration file are change to 3.9.4.001 release